### PR TITLE
Fix screenshot backend

### PR DESCRIPTION
**Pull request by AndrewSouthpaw**
Update Heroku, Cloudinary, and server backend to make it all work

### CODE REVIEW

File: server/api/screenshot/screenshotController.js [hunk 1]
```diff
@@ -15,33 +15,35 @@ exports.list = function(req, res, next){
...
+/**
+ * create
+ * ======
+ * Takes screenshot of specified URL, creates new Screenshot in DB, and
+ * adds screenshot to the user images.
+ */
 exports.create = function(req, res, next) {
-  var url = req.body.url;
   var username = req.params.username;
-  var annotatedImage = req.body.annotatedImage;
```
Comment (ruben1) [inline server/api/screenshot/screenshotController.js:21]: if we remove annotatedImage variable, we should remove it also from the Screenshot creation on line 27 and also remove it from being required on the db

**Comment by ruben1** [comment]
besides the previous comment, i think its ok

File: server/api/screenshot/screenshotController.js (hunk 1 above)
Comment (AndrewSouthpaw) [inline server/api/screenshot/screenshotController.js:21]: Right now I'm putting the imageUrl from the takeScreenshot into both `originalImage` and `annotatedImage`. It'll be something like a placeholder until we add that feature. Seem reasonable?

Comment (ruben1) [inline server/api/screenshot/screenshotController.js:21]: yes, I was thinking something similar

**Pull request merged**